src/successModal/SuccessModal.tsx

```diff
@@ -37,15 +38,14 @@ class SuccessModal extends Component<SuccessModalProps> {
       forwardedRef,
       imageSrc,
       imageText,
-      ariaLabelledBy,
-      ariaDescribedBy,
```

Comment:
Breaking change?

Reply:
No previously those props were not well used and not defined in the render at the end.